Source/DiabloUI/ui_item.h

```diff
@@ -42,36 +43,60 @@ enum UiFlags : uint16_t {
 	// clang-format on
 };
 
+inline UiFlags operator|(UiFlags lhs, UiFlags rhs)
+{
+	using T = std::underlying_type_t<UiFlags>;
+	return static_cast<UiFlags>(static_cast<T>(lhs) | static_cast<T>(rhs));
+}
+
+inline UiFlags operator|=(UiFlags &lhs, UiFlags rhs)
+{
+	lhs = lhs | rhs;
+	return lhs;
+}
+
+inline UiFlags operator&(UiFlags lhs, UiFlags rhs)
+{
+	using T = std::underlying_type_t<UiFlags>;
+	return static_cast<UiFlags>(static_cast<T>(lhs) & static_cast<T>(rhs));
+}
+
+inline UiFlags operator&=(UiFlags &lhs, UiFlags rhs)
+{
+	lhs = lhs & rhs;
+	return lhs;
+}
+
+inline UiFlags operator~(UiFlags value)
+{
+	using T = std::underlying_type_t<UiFlags>;
+	return static_cast<UiFlags>(~static_cast<T>(value));
+}
+
+inline bool hasAnyOf(UiFlags lhs, UiFlags test)

```

Comment:
Not sure why you decided to do this with a lower case since the style guide says functions start with an upper case :)

Reply:
I blame writing code while eating breakfast :D. I'll update that now.